docs/ui/organizations.md

```diff
@@ -21,15 +22,41 @@ For larger teams or companies with more complex needs around user and access man
 
 See the [Prefect Cloud plans](https://www.prefect.io/pricing) to learn more about options for supporting more users, service accounts, and workspaces.
 
+## Organizations overview
+
+An organization is a type of account available on Prefect Cloud that enables more extensive and granular control over inviting workspace collaboration. Organizations are only available on Prefect Cloud.
+
+Within an organization account you can:
+
+- Invite members to join the organization.
+- Give members administrator or collaborator roles within the organization.
```

Comment:
Users can be collaborators in a workspace, but not within an organization.

Reply:
```suggestion
- Give members administrator or member roles within the organization.
```